### ISSUE TITLE
Move to Bootstrap v5.*

### ISSUE THREAD

**Issue by 2ynn** [LesFruitsDefendus/saskatoon-ng#112]
Most notably what I feel is really limiting me:
- column offsets : e.g. `col-offset-md-3` : https://getbootstrap.com/docs/4.0/layout/grid/#offsetting-columns
- proper margin/padding: e.g. `md-x-4` https://getbootstrap.com/docs/5.1/utilities/spacing/
- horizontal alignment: e.g. `justify-content-md-center` : https://getbootstrap.com/docs/5.0/layout/columns/#horizontal-alignment

**Comment by farhanmasud** [comment]
The current saskatoon-ng [uses Bootstrap v3.3.6](https://github.com/LesFruitsDefendus/saskatoon-ng/blob/develop/saskatoon/sitebase/static/css/bootstrap.min.css). So, the process will be migrating Bootstrap v3.3.6 to v5.*.

What will be the best way of approaching this? Starting with the base template located in saskatoon/sitebase/templates/app/base directory (referring to the develop branch) and then updating other templates one by one?

I have [found tests](https://github.com/LesFruitsDefendus/saskatoon-ng/blob/develop/saskatoon/tests/helpers.py) that relies on Bootstrap classes to locate element so in the process those will have be updated.

**Comment by 2ynn** [comment]
This is a big one. I will flag this issue as low priority for now.
Running diagnostic tests would be nice, if you do so please use the develop branch as I have tried to reorganize templates in a more generic way to avoid layout duplication.

**Comment by farhanmasud** [comment]
@2ynn thanks for your feedback. I'll look into it and check what I can do about it.

**Comment by farhanmasud** [comment]
Okay, I'll look into other issues to fix now

**Comment by Superpat** [comment]
Yeah this is probably almost as big a job as moving to tailwind or building our own css. In any case, I would recommend that we build our own css rules instead of using a framework if we ever get to this.
